app/models/concerns/questionable.rb

```diff
@@ -89,5 +89,9 @@ def mandatory_questions
     def all_questions_answered?
       mandatory_questions.all?(&:answered?)
     end
+
+    def any_questions_answered?
```

Comment:
What about?:

```
   def no_questions_answered?		
      mandatory_questions.none?(&:answered?)		
    end
```

Reply:
Have refactored the body of the method to more closely match your example but kept the name the same. Calling `any_questions_answered?` from the view is more readable than `!no_questions_answered?`.